action.yml

```diff
@@ -9,8 +9,13 @@ inputs:
     description: 'The type of comment. Options: [fmt, init, plan]'
     required: true
   commenter_input:
-    description: 'The comment to post from a previous step output'
-    required: true
+    description: 'the comment to post from a previous step output'
+    required: false
+    default: ''
+  commenter_input_file:
+    description: 'the comment to post from a previous step output passed as file name (located in )'
```

Comment:
there seems to be a word missing

Reply:
Yes I had `{{ github.workspace }}` in there but it leads to an error when executing the action as it seems to be replaced.